rootfs/api/models/app.py

```diff
@@ -393,13 +393,18 @@ def _default_structure(self, release):
         if not release.build.sha:
             structure = {'cmd': 1}
 
-        # if a dockerfile exists without a procfile, assume docker workflow
-        elif release.build.dockerfile and not release.build.procfile:
+        elif release.build.procfile and 'web' in release.build.procfile:
```

Comment:
no..i am checking whether procfile has web before checking the docker file so it would create a type of web